src/cuga/backend/cuga_graph/nodes/cuga_flow/flow_agent.py

```
@@ -0,0 +1,496 @@
+        if agent is None:
+            return flows[0].id if flows else ""
+        try:
+            flow_id = await agent.route(flows, ctx.current_state)
+            tracker.collect_step(
+                Step(
+                    name=f"Gateway {ctx.element_name or gateway_id}",
+                    data=f"Routed to flow: {flow_id}",
+                )
+            )
+            return flow_id
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)